Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.19-alpine as builder
```

Comment:
golang is at 1.20 now. Can we use that?

Reply:
I used 1.20 locally, so it should work.  I was just using 1.19 because it matched https://github.com/cerc-io/eth-statediff-service.